### PR TITLE
Fix formatting for gradient path with several params

### PR DESCRIPTION
**Pull request by egorikftp**

### CODE REVIEW

File: components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg
Comment (Goooler) [inline components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg:1]: Would prefer the SVG file from https://kotlinlang.org/docs/kotlin-brand-assets.html#kotlin-logo

File: components/generator/common/src/main/kotlin/io/github/composegears/valkyrie/generator/ext/CodeBlock.kt [hunk 1]
```diff
@@ -2,8 +2,35 @@ package io.github.composegears.valkyrie.generator.ext
...
+    argumentFlow: String,
+    vararg args: Any?,
+    block: CodeBlock.Builder.() -> Unit,
+) {
+    add("$argumentFlow\n", *args)
+    indention(block)
+    newLine()
+    add("),")
+    newLine()
+}
```
Comment (Goooler) [inline components/generator/common/src/main/kotlin/io/github/composegears/valkyrie/generator/ext/CodeBlock.kt:26]:
```suggestion
fun CodeBlock.Builder.argumentBlock(
    argumentFlow: String,
    vararg args: Any?,
    isNested: Boolean = false,
    block: CodeBlock.Builder.() -> Unit,
) {
    add("$argumentFlow\n", *args)
    indention(block)
    newLine()
    add(")")
    if (isNested) {
        add(",")
        newLine()
    }
}
```

**Review by Goooler — APPROVED** [review]

File: components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg
Comment (egorikftp) [reply components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg:1]: Will check if this SVG contains my case)

Comment (egorikftp) [reply components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg:1]: Checked, provided icon covered by `ic_linear_gradient.svg` case

Comment (egorikftp) [reply components/generator/imagevector/src/test/resources/ic_linear_gradient_with_stroke.svg:1]: Will use official kotlin logo in next feature 🙂

**Pull request merged**